numpy/linalg/lapack_lite/f2c_config.c

```diff
@@ -1773,7 +1759,7 @@ doublereal slamch_(char *cmach)
 doublereal slamc3_(real *a, real *b)
 {
     /* System generated locals */
-    volatile real ret_val;
+    real ret_val;
```

Comment:
Is this change deliberate?

Reply:
This was introduced during regeneration. The variable is declared locally and isn't returned before the end of the function, so I personally can't see a reason why this routine wouldn't be subject to optimization.

Reply:
I'm a little concerned that the regeneration produced a different result. What's your F2C version and lapack source version?

Reply:
```
f2c (Fortran to C Translator) version 20100827.
```

LAPACK version is the one specified in the README (3.2.2 I think).

Reply:
That's the same version as mine...

Reply:
Well, I rebuilt with my python 2 setup and this didn't change, so I have no idea what's going on, but I guess it's ok.

Reply:
Note: I did not compile f2c myself. I tried to compile a 64-bit version with anaconda, but there was a fault, so I gave up and downloaded the binary from netlib.